config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -229,6 +229,22 @@ presubmits:
           - e2e_test_gke.yaml
           - --test_result_folder
           - e2e_test
+    - name: presubmit-e2e-test-gke
```

Comment:
Can you follow the convention of the rest of the file and rename pipelines tests to 
<org>-<repo>-<tag>

Reply:
This test case is just a copy-paste of the existing `presubmit-e2e-test` entry.
I do not do modification in-place to ensure that nothing breaks.

Reply:
I know but is there a reason not to follow the naming convention? The existing tests should probably be following the existing convention as well. I'm guessing I reviewed the original PR adding the pipelines tests and missed this.